### ISSUE TITLE
Search input - focus bug

### ISSUE THREAD

**Issue by claudiac2024** [sl-design-system/components#1790]
### Provide a general summary of the issue here

Developers from Neoris have found issues regarding the search input component. We have the search field in the table of content component.

<img width="1438" alt="Image" src="https://github.com/user-attachments/assets/2a4bbaf2-3ca1-4bc3-82b3-18b261e6a408" />

The "close x" button is not focusable, and the icon and target are smaller than 24px.

### 🤔 Expected Behavior?

The user should be able to focus the X button by using tab. 

### 😯 Current Behavior

Add property tabindex="-1" to avoid clear button being focused, as Accessibility documentation states that it should only be activated when pressing ESC key, and TAB should skip this button:

Esc Clear search value, when is an input 

Upscale xmark icon's size from 16x16 to 24x24 to improve accesibility.
Change aria-label value to "Clear" instead of "Clear text" because of voice control.

### 💁 Possible Solution

Seems like in aria-label=$msg('Clear text'), $msg() is only a translating function, but does not give us the possibility to change or customize the message. Devs updated the list to request the change from "Clear text" to "Clear" as Zenobia recommended.

### 🖥️ Steps to Reproduce

[https://libromediaviewer6.web.app/](url)

staging space from devs

### To do

- [ ] Investigate, come up with a solution 

### What browsers are you seeing the problem on? (only for bugs in code)

Chrome

### What operating system are you using?

iOS

### 👤 Your name

Claudia Canales

### 🧢 Your product/team

Neon, My digital book, LMV, AV4 

### 🎨 Your Theme(s)

Sanoma Learning

**Comment by Diaan** [comment]
https://designsystems.surf/components/search

**Comment by jpzwarte** [comment]
Here's some good input for this discussion: https://scottaohara.github.io/clear-text-field-button/
